src/Workleap.DomainEventPropagation.Analyzers/EventDomainAttributeUsageAnalyzer.cs

```diff
@@ -0,0 +1,94 @@
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Workleap.DomainEventPropagation.Analyzers.Internals;
+using Workleap.Extensions.MediatR.Analyzers.Internals;
+
+namespace Workleap.DomainEventPropagation.Analyzers;
+
+[DiagnosticAnalyzer(LanguageNames.CSharp)]
+public sealed class EventDomainAttributeUsageAnalyzer : DiagnosticAnalyzer
```

Comment:
Did we test the analyzer performance like described here?
https://www.meziantou.net/understanding-the-impact-of-roslyn-analyzers-on-the-build-time.htm

Reply:
No, will do in the next iteration (with the other rule)